src/main/kotlin/org/jetbrains/research/testspark/display/TestSparkDisplayBuilder.kt

```diff
@@ -41,6 +41,11 @@ class TestSparkDisplayBuilder {
         toolWindow!!.show()
 
         generatedTestsTabBuilder!!.getRemoveAllButton().addActionListener {
+            if (generatedTestsTabBuilder!!.getGeneratedTestsTabData().testCaseNameToPanel.isEmpty()) {
+                clear()
+                return@addActionListener
+            }
```

Comment:
I do not understand what exactly this if-statement does. Could you explain (and mention it in the comment above the if-statement as well)?